Automattic/jetpack
# Admin Menus: Fix users menu item pointing to wp-admin.

Fixes p1688556726242369-slack-CBG1CP4EN

## Proposed changes:

* Fixes users menu item pointing to the wp-admin version.

The PR https://github.com/Automattic/jetpack/pull/31170/ introduced a regression where the class `Atomic_Admin_Menu` extended the `add_users_menu` function from [Admin_Menu::add_users_menu](https://github.com/Automattic/jetpack/blob/3e036936f2385aaef046a5181d08ed263ff1c456/projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php#L327-L335) but did not call the original parent function to run execute additional menu configuration.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Create a new Business site, mark it as a WoA Developer Blog, and take it Atomic.
* Install the Jetpack Beta Tester plugin and check out this branch on the site.
* Clicking on the Users menu should go to Calypso `/people/team`.
